stub/src/main/java/io/grpc/stub/ClientCalls.java

```diff
@@ -115,7 +114,7 @@ private ClientCalls() {}
       return getUnchecked(futureUnaryCall(call, param));
     } catch (Throwable t) {
       call.cancel();
-      throw Throwables.propagate(t);
+      throw t instanceof RuntimeException ? (RuntimeException) t : new RuntimeException(t);
```

Comment:
I put this in here to contrast the top  version.  

Functionally I believe they are equivalent, mostly I put them in here for readability concerns.